site-content/source/modules/ROOT/pages/development/how_to_commit.adoc

```diff
@@ -14,35 +14,43 @@ Hypothetical CASSANDRA-12345 ticket is a cassandra-4.0 based bug fix
 that requires different code for cassandra-4.0, cassandra-4.1, and
 trunk. Contributor Jackie supplied a patch for the root branch
 (12345-4.0.patch), and patches for the remaining branches
-(12345-4.1.patch, 12345-trunk.patch).
+(12345-4.1.patch, 12345-5.0.patch, 12345-trunk.patch).
 
 On cassandra-4.0:::
   . `+git am -3 12345-4.0.patch+` (any problem b/c of CHANGES.txt not
   merging anymore, fix it in place)
-  . `+ant realclean && ant jar build-test+` (rebuild to make sure code
+  . `+ant realclean && ant jar build-test check+` (rebuild to make sure code
```

Comment:
all of the occurrences of `jar build-test` are superfluous, as jar already calls build-test

i.e. these can all be `ant realclean && ant jar check`

Reply:
Actually I just realized I made a mistake, check should be added only for the 5.0 and trunk branches.
I will remove build-test, thanks